### ISSUE TITLE
нужно бот в телеграм создавать для поднятия вакансии?

### ISSUE THREAD

**Issue by m6129** [sergo-code/hh-ru-auto-resume-raising#10]
![image](https://github.com/user-attachments/assets/80c03ce8-8f74-41ec-ac84-60c8b0be13d5)

**Comment by thunderom** [comment]
Да, при создании бота через BotFather ты получишь токен, его, соответственно, нужно будет вписать сюда:
bot_token=

А в поле "admin_tg=" тебе нужно будет указать свой id из tg, его можно узнать в боте @userinfobot
